src/useAudioContext.tsx

```diff
@@ -73,7 +71,10 @@ export function useAudioContext<S extends string>(
   props: Props<S>,
 ): UseAudioContext<S> | null {
   const [soundEffectVolume] = useSetting(soundEffectVolumeSetting);
-  const { audioOutput } = useMediaDevices();
+  const devices = useMediaDevices();
+  const audioOutputId = useObservableEagerState(
+    devices.audioOutput.selected$,
+  )?.id;
```

Comment:
This would also work right?
I have seen the pattern, that we wrap hooks into hooks more often recently so this would not be a code style inconsistency.
```suggestion
  const audioOutputId = useObservableEagerState(
    useMediaDevices().audioOutput.selected$,
  )?.id;
```